File: 백준/Bronze/2309. 일곱 난쟁이/일곱 난쟁이.py
lst = []
for _ in range(9):
    lst.append(int(input()))


for i in range(9): 
    for j in range(i+1, 9):
        if sum(lst) - (lst[i] + lst[j]) == 100:
            a = lst[i]
            b = lst[j]

lst.sort()
for l in lst:
    if l == a or l == b:
        continue
    print(l)

File: 백준/Bronze/2309. 일곱 난쟁이/일곱 난쟁이.js
const fs = require("fs");
const arr = fs
  .readFileSync("/dev/stdin")
  .toString()
  .trim()
  .split("\n")
  .map(Number);

const lst = arr.reduce((acc, val) => acc + val, 0);
let a;
let b;
for (let i = 0; i < arr.length; i++) {
  for (let j = 1; j < arr.length; j++) {
    if (lst - (arr[i] + arr[j]) === 100) {
      a = arr[i];
      b = arr[j];
    }
  }
}

arr.sort((a, b) => a - b);
for (let k of arr) {
  if (k === a || k === b) continue;
  console.log(k);
}
